test/domains/compilerErrors/notsupportedStridable-assoc.good

```diff
@@ -0,0 +1 @@
+notsupportedStridable-assoc.chpl:3: error: associative domains do not support .stridable
```

Comment:
Do we want to edit this error message now to make it consistent with your changes, or do it as a followup? I was expecting `this domain type does not support 'stridable'`.

Reply:
This error is issued by already-existing code. I don't think we need to change it.